vertx-pg-client/src/test/java/io/vertx/pgclient/PgPoolTest.java

```diff
@@ -162,4 +179,43 @@ public void testUseAvailableResources(TestContext ctx) {
       pool.close();
     }
   }
+
+  // Scenario:
+  // When a user gets a connection from pgPool some milliseconds behind IDLE expiration time,
+  // sometimes you get the following error:
+  // Fail to read any response from the server, the underlying connection might get lost unexpectedly.
+  @Test(expected = TimeoutException.class)
+  public void checkBorderConditionBetweenIdleAndGetConnection(TestContext ctx) {
```

Comment:
This test must be moved to the abstract pool test class because the timeout features applies to all connection pools (not just Pg)